wp-content/themes/citylimits/inc/widgets/class-citylimits-special-projects-featured-content-widget.php

```diff
@@ -151,6 +151,7 @@ function widget( $args, $instance ) {
 			// close the ul
 			echo '</ul>';
 
+			echo '</div>';
```

Comment:
Where does this come from?

Reply:
Good question. Removed in https://github.com/INN/umbrella-citylimits/pull/106/commits/92a750e1acf91d89d8727d25121f2a2cfa36dd61